include/deal.II/fe/fe_q.h

```diff
@@ -559,11 +559,16 @@ class FE_Q : public FE_Q_Base<dim, spacedim>
    * Constructor for tensor product polynomials with support points @p points
    * based on a one-dimensional quadrature formula. The degree of the finite
    * element is <tt>points.size()-1</tt>. Note that the first point has to be
-   * 0 and the last one 1. Constructing
+   * 0 and the last one 1, to ensure that the resulting element is continuous.
+   * As a consequence, only *certain* quadrature formulas make sense as
+   * argument to this constructor. Specifically, constructing
    * <tt>FE_Q<dim>(QGaussLobatto<1>(fe_degree+1))</tt> is equivalent to the
    * constructor that specifies the polynomial degree only. For selecting
-   * equidistant nodes at <tt>fe_degree > 2</tt>, construct
-   * <tt>FE_Q<dim>(QIterated<1>(QTrapezoid<1>(),fe_degree))</tt>.
+   * equidistant nodes at <tt>fe_degree > 2</tt>, use
+   * <tt>FE_Q<dim>(QIterated<1>(QTrapezoid<1>(),fe_degree))</tt>, though the
+   * use of equidistant node points is known to lead to poorly conditioned
+   * linear systems with poor approximation properties for high polynomial
```

Comment:
I am not sure if we should be more specific here. The *approximation properties* should technically not be affected, as the Galerkin procedure of typical finite element algorithms picks the best fit among the polynomials represented in the space, irrespective the actual representation in the basis. The problem is the interpolation properties that might spoil things (as everyone learns for high-order Lagrange interpolation in a first-year numerical analysis course) and the conditioning aspect you mention above, possibly leading to larger round-off errors. I would therefore tend towards writing
```suggestion
   * linear systems with poor interpolation properties for high polynomial
```
(but I am happy to be convinced otherwise).